server/src/main/java/org/elasticsearch/search/fetch/FetchContext.java

```diff
@@ -69,13 +68,6 @@ public ContextIndexSearcher searcher() {
         return searchContext.searcher();
     }
 
-    /**
-     * The mapper service for the index we are fetching documents from
-     */
-    public MapperService mapperService() {
-        return searchContext.mapperService();
-    }
-
```

Comment:
🎆

Reply:
deja vu? I certainly made this change twice :)